# Service to start/stop recording
#
# Fields
# ------
# enable_recording : Whether to (re)start or stop recording
# stop_recording   : If stopping, the recorded evaluation result

bool enable_recording
---
float64 evaluation